search-app/src/cmr/search/data/query_to_elastic.clj

```diff
@@ -277,6 +281,10 @@
   [_]
   {:tag-key :tag-key.lowercase})
 
+(defmethod q2e/concept-type->sort-key-map :variable
+  [_]
+  {:variable-name :variable-name.lowercase})
```

Comment:
This came up in a PR from Lauren yesterday: the KEY.QUALIFIER convention ... is this an elasticsearch-ism? (i.e., why don't we use `variable-name-lowercase`?)